packages/dom-expressions/src/server.js

```diff
@@ -674,3 +693,33 @@ export function ssrSpread(props, isSVG, skipChildren) {
   }
   return result;
 }
+
+// client-only APIs
+
+export {
+  notSup as classList,
+  notSup as style,
+  notSup as insert,
+  notSup as spread,
+  notSup as delegateEvents,
+  notSup as dynamicProperty,
+  notSup as setAttribute,
+  notSup as setAttributeNS,
+  notSup as addEventListener,
+  notSup as render,
+  notSup as template,
+  notSup as setProperty,
+  notSup as className,
+  notSup as assign,
+  notSup as hydrate,
+  notSup as getNextElement,
+  notSup as getNextMatch,
+  notSup as getNextMarker,
+  notSup as runHydrationEvents
+};
```

Comment:
And are these accurate? Wondering if I missed an export from somewhere else that does in fact work server-side and should not be mapped to `notSup`.

Reply:
These are all new functions never exported before for server, so it is non-breaking to make them errors I believe. However the question is, do you want any of them to be no-ops instead of error-throwing? I think errors are better for DX to inform people what not to do.

Also, any of these can be overriden to be no-ops over in solid-js, for example like is done here:

https://github.com/solidjs/solid/pull/2269/files#r1740197787